recipes/bytecode/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "bytecode" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f78a312880173f1d20aef2eceb8eb959dc6c2167966b757fef201b8af839f802
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
I think this recipe can be `noarch`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages. Thus, please remove the `skip` line and add `noarch: python` instead.